api/v1alpha1/dspipeline_types.go

```diff
@@ -122,6 +122,9 @@ type MlPipelineUI struct {
 type Database struct {
 	*MariaDB    `json:"mariaDB,omitempty"`
 	*ExternalDB `json:"externalDB,omitempty"`
+	// +kubebuilder:default:=false
+	// +kubebuilder:validation:Optional
+	DisableHealthCheck bool `json:"disableHealthCheck"`
```

Comment:
can you update this yaml with the addition of these [new fields](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/samples/dspa_all_fields.yaml):